### ISSUE TITLE
Understanding the landscape of Open Life Science in India '22

### ISSUE THREAD

**Issue by nilabha1512** [open-life-science/ols-6#22]
Project Lead: Nilabha 'Rik' Mukherjea
Project Mentor: Anne Treasure

***
Welcome to OLS-6! This issue will be used to track your project and progress during the program. Please use this checklist over the next few weeks as you start the Open Life Science program :tada:.

***

## Week 1: Meet your mentor!
- [x] Meet mentor for 30 minutes 
- [x] Create an account on [GitHub](https://github.com)
- [x] Check if you have access to the HackMD notes set up for your meetings with your mentor
- [x] Prepare to meet your mentor(s) by completing a short homework provided in your shared notes
- [x] Complete **your own copy** of the [open leadership self-assessment](https://docs.google.com/document/d/1oQgdfj4lPnypAyb9_Ba0Zt7E8J5L6qMvuKwu0wgQsjs/edit?usp=sharing) and share it to your mentor 
  If you're a group, each teammate should complete this assessment individually. This is here to help you set your own goals during the program. No need to share your results, but be ready to share your thoughts with your mentor.
- [x] Make sure you know when and how you'll be meeting with your mentor.

## Week 2 - Cohort Call (Welcome to Open Life Science!)

- [x] Attend call or catch up via YouTube
- [x] Create an issue on the [OLS-6 GitHub repository](https://github.com/open-life-science/OLS-6/issues/new) for your OLS work and share the link with your mentor.
- [x] Draft a brief vision statement using your goals

  [This lesson](https://mozilla.github.io/open-leadership-training-series/articles/introduction-to-open-leadership/stating-your-project-vision/) from the Open Leadership Training Series (OLTS) might be helpful

- [ ] Leave a comment on this issue with your draft vision statement & be ready to share this on the call
- [ ] Check the [Syllabus](https://openlifesci.org/ols-6) for notes and connection info for all the cohort calls.

## Week 3: Meet your mentor!

- [x] Meet mentor
- [ ] Look up two other projects and comment on their issues with feedback on their vision statement
- [ ] Complete this [compare and contrast assignment](https://docs.google.com/document/d/1ukvqDRIYfvCapVMdE5hWP-0MkLNJ9T65X43O7F336Ac/edit?usp=sharing) about current and desired community interactions and value exchanges
- [x] Complete your Open Canvas ([instructions](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas/), [canvas](https://docs.google.com/presentation/d/1MeJo0TyuMg_waLk1J4q9y1aAqKNMuRBlnmxEChSz-cQ/edit?usp=sharing))
- [x] Share a link to your Open Canvas in your GitHub issue
- [ ] Start your [Roadmap](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/start-your-project-roadmap/)
- [ ] Comment on your issue with your draft Roadmap
- [ ] Suggest a cohort name at the bottom of the shared notes and vote on your favorite with a +1

## Week 4: Cohort Call (Tooling and road mapping for Open projects)

- [x] Attend call or catch up via YouTube
- [ ] Look up two other projects and comment on their issues with feedback on their open canvas.

## Week 5 and later
- [x] Meet mentor
- [x] Create a GitHub repository for your project
- [x] Add the link to your repository in your issue
- [ ] Use your canvas to [start writing a `README.md` file](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/), or landing page, for your project
- [ ] Link to your README in a comment on this issue
- [ ] Add an [open license](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/sharing-your-work-in-the-open/) to your repository as a file called `LICENSE.md`
- [ ] Add a [Code of Conduct](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-a-code-of-conduct/) to your repository as a file called `CODE_OF_CONDUCT.md`
- [ ] Invite new contributors to your work!

This issue is here to help you keep track of work as you start the Open Life Science program. Please refer to the [OLS-6 Syllabus](https://openlifesci.org/OLS-6) for more detailed weekly notes and assignments past week 4.

## Week 6 
- [ ] Attend call or catch up via YouTube

## Week 7
- [x] Meet mentor

## Week 8
- [ ] Attend call or catch up via YouTube

## Week 9
- [ ] Meet mentor

## Week 10
- [ ] Attend call or catch up via YouTube

## Week 11
- [ ] Meet mentor

## Week 12
- [ ] Attend call or catch up via YouTube

## Week 13
- [ ] Meet mentor

## Week 14
- [ ] Attend call or catch up via YouTube

## Week 15
- [ ] Meet mentor

**Comment by harisood** [comment]
Seconded, very curious @nilabha1512 ! Do you have your vision statement yet?

**Comment by nilabha1512** [comment]
## **OPEN CANVAS** 
[https://www.canva.com/design/DAFPZl5VfX0/view?utm_content=DAFPZl5VfX0&utm_campaign=designshare&utm_medium=link&utm_source=publishsharelink](url)

![Open Canvas  OLS Cohort VI  Nilabha 'Rik' Mukherjea](https://user-images.githubusercontent.com/80630656/202880513-65d84b74-4fb4-4609-beb8-847339a8ad0b.png)

**Comment by nilabha1512** [comment]
## **VISION STATEMENT**

### **TLDR (To Long Didn't Read):  I wish to realize an India where research and the concepts of open science are encouraged and championed from the get-go.**

I wish to realize an India where research and the concepts of open science are encouraged and championed from the get-go. 

This is to enable future generations of life science enthusiasts to find the right avenue of research in both academia and industry through informed decisions. I truly believe that exploring what is out there today, can help us be there and support future initiatives from younger generations more actively and competently. 

## **Who am I working with?**

Right now, I am working alone with help of my mentor, Anne Treasure. In the future, I will be open to joining forces with like-minded and passionate individuals who are crazy enough to believe that they can change the world. 

## **What am I doing?**

Past. Present. Future. I am trying to understand the Past, to define the Present. I am defining the Present to help realize what the Future could be. To understand the Past, I am working through the existing literature on research practices, open science initiatives, and existing government policies for the same. For the Present, I am talking with my professors and connections in both academia and the industry to understand why there remains a sense of mistrust between these two axes. Finally, for the Future, I am trying to visualize on paper which nature of this future would suit my fellow undergraduates and future generations. 

## **Who am I doing this for?**

1. For my peers, colleagues, professors and industry professionals, and future Nilabhas who want to know if the uncharted can be charted.
2. For myself, I never want to feel helpless and uncertain about my future. My education and exposure to different avenues of life require me to be a responsible student and global citizen and attempt to realize the landscape of research in my home country. 

This is my vision. My dream is for a better world. Right now, I do not know how to make it, happen. But I will make sure to give my best. Your opinions and strategies to make it happen, are more than welcome. 

P.S. I will make it a point to be more active here. If you wish to get updates, do share your email as a comment. : )

**Comment by nilabha1512** [comment]
## **ROADMAP**
https://github.com/nilabha1512/OLS-Cohort-VI/issues/3#issue-1456830033

**Comment by babasaraki** [comment]
Impressive work!

**Comment by harinilakshminarayanan** [comment]
@nilabha1512 Your vision statement and the roadmap to get there is really very impressive. I am glad to see this project developing and would like to get updates. I am also happy to help if there is some way for me to do so
harini.lakshminarayanan@usz.ch
